### PR TITLE
add escape and write commands

### PR DESCRIPTION
**Pull request by vtm9**

### CODE REVIEW

File: plugin/pry.vim [hunk 1]
```diff
@@ -16,6 +16,8 @@ function! pry#insert()
     let text = get(g:pry_map, &filetype)
     call feedkeys('o', 'i')
     call feedkeys(text)
+    call feedkeys("\<Esc>")
+    call feedkeys(":w\<CR>")
```
Comment (BlakeWilliams) [inline plugin/pry.vim]: I like the `<Esc>` but I don't think writing is automatically is a good practice. Can we remove that?

**Comment by BlakeWilliams** [comment]
1 minor thing, otherwise LGTM

File: plugin/pry.vim (hunk 1 above)
Comment (vtm9) [reply plugin/pry.vim]: Ok

**Comment by vtm9** [comment]
Hi. Is there any reason to not merge it?)

**Comment by BlakeWilliams** [comment]
Just busy is all. :)

**Pull request merged**